src/main/java/com/avispl/symphony/dal/communicator/aggregator/ZoomRoomsAggregatorCommunicator.java

```diff
@@ -290,7 +290,7 @@ public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttp
     private static final long defaultRoomDevicesTimeout = 60 * 1000 / 2;
 

```

Comment:
sorry, Github does not let me comment on top of class, so adding comment here. Please make the Copyright statement:

```
/*
 * Copyright (c) 2021-2022 AVI-SPL, Inc. All Rights Reserved.
 */
```